Makefile

```diff
@@ -836,6 +836,13 @@ trwiki_models: \
 trwiki_tuning_reports: \
 	tuning_reports/trwiki.wp10.md
 
+####################### Ukrainian Wikipedia #################################
+datasets/ukwiki.labelings.20200501.json:
+	./utility extract_labelings \
+		/mnt/data/xmldatadumps/public/ukwiki/20200501/ukwiki-20200501-pages-meta-history?*.xml-*.bz2 \
```

Comment:
Is the `?` after `history` a typo?

Reply:
It ensure that we match `ukwiki-20200501-pages-meta-history1.xml-*.bz2` but that we don't match `ukwiki-20200501-pages-meta-history.xml-*.bz2`.  The later sometimes appears as a single file full of the entire history.